### DISCUSSION TITLE
Outside banked corner for Hybrid coaster

### DISCUSSION THREAD

**Discussion by DoeEensGek** [OpenRCT2/OpenRCT2#18695] [Feature requests]
Is it possible to create an outside banked corner for the RMC hybrid coaster?
Untamed in Walibi Holland contains one at the beginning, shown in the link under.
https://www.themeparkarchive.com/Archive/ExtraLarge/53283.jpg

**Comment by karst** [comment]
@X123M3-256 has made a mockup of this before, but it wasn't functional. Would be pretty cool to see though.

**Reply by X123M3-256** [reply]
You mean this? https://i.imgur.com/vSGuAxd.png Those pieces proved to be pretty much useless ... outward banked turns never look like this. I do not plan to ever implement standard banked turn pieces with reversed banking like this - I would make the outward banked turns a special piece like overbanks.

**Reply by karst** [reply]
Yep those, they're only available on Untamed and on Project Exodus as of right now, and in both cases directly out of the station.

**Reply by X123M3-256** [reply]
Even the one on Untamed doesn't look like this, it starts level and banks as it turns. Most outer banks are combined with an airtime hill and the few that aren't are always taken slowly, because otherwise it'd produce uncomfortably high laterals.

The standard turn pieces with reversed banking proved to be pretty much useless for building anything resembling an actual outer bank. The untamed style turn looks OK I guess but that's really the only use I could find for them and it still isn't right. There are better ways to do this.

**Reply by DoeEensGek** [reply]
> Even the one on Untamed doesn't look like this, it starts level and banks as it turns. Most outer banks are combined with an airtime hill and the few that aren't are always taken slowly, because otherwise it'd produce uncomfortably high laterals.
> 
> The standard turn pieces with reversed banking proved to be pretty much useless for building anything resembling an actual outer bank. The untamed style turn looks OK I guess but that's really the only use I could find for them and it still isn't right. There are better ways to do this.

As you stated, the one on Untamed start flat and banks while turning, while after a 90-degree turn it goes back to level. Is your intention to create one 180 degree piece, or is two 90 degree pieces?
A standard turn(https://i.postimg.cc/L8jPzYyK/Mel-s-World-2022-11-30-11-40-19.jpg) starting from flat ending banked, just like going level to banked with a straight piece, an option (https://i.postimg.cc/m2tNFrfB/Mel-s-World-2022-11-30-11-39-17.jpg)?. That would make the corner available in building in the default building window and not needed to put in special part list.

**Comment by ocalhoun6** [comment]
Would be amazing to have, yeah, but it's not easy. In addition to making the track pieces from scratch, the ride building window would need to have some work done to make it actually buildable. Might even need some tweaking in the ride physics to properly calculate the g-forces of an outside banked turn.

**Reply by karst** [reply]
The UI wouldn't be so difficult. It would just have the bank on and allow the turns to go both ways.

**Reply by X123M3-256** [reply]
Changes to the ride building window aren't that hard to make, and not really necessary if you just put it in the special track piece menu (I'd prefer to avoid cluttering the menu by putting everything there though - I'd prefer a different interface for overbanks and outward banks and I've got some ideas).

Implementing G force code is something you need to do for almost every nontrivial track piece. As track pieces go I don't think this one is particularly hard from a technical standpoint, but deciding exactly what shape the piece(s) should be so as to be most useful, that's tricky.

**Comment by DoeEensGek** [comment]
Since there are some cool new updates for a few coasters recently, let's give this a small kick.
